BetterCallSodium/External/MonoPInvokeCallbackAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma warning disable IDE0052
```

Comment:
Is it 100% necessary for the `Type` to be private? Just making it `public` would make sense to me _and_ remove the need to suppress a warning.

Reply:
The code was taken from external sources, I don't want to do many changes to it.